plugins/doc_fragments/module_certificate.py

```diff
@@ -451,8 +451,8 @@ class ModuleDocFragment(object):
             - Time will always be interpreted as UTC.
             - Valid format is C([+-]timespec | ASN.1 TIME) where timespec can be an integer
               + C([w | d | h | m | s]) (e.g. C(+32w1d2h).
-            - Note that if using relative time this module is NOT idempotent.
             - If this value is not specified, the certificate will start being valid from now.
+            - Note that this value is B(not used for idempotency checks).
```

Comment:
Is the phrase "idempotency checks" commonly used throughout the Ansible literature? We understand the consequences of this, but for an end user it might be better to indicate that changes to this value will not cause certificates to be regenerated.

Reply:
Good question. What what you change it to, something like `Note that this value is B(not used to determine whether the certificate should be regenerated).`?

Reply:
I guess to remove any ambiguity how about "Note that this value is B(not used to determine whether __an existing__ certificate should be regenerated)."

Reply:
Adjusted in 00cdc994494db9891fc5ca4a554adcefebdd6a6e.